internal/api/external.go

```diff
@@ -635,7 +634,7 @@ func getErrorQueryString(err error, errorID string, log logrus.FieldLogger, q ur
 			log.WithError(e.Cause()).Info(e.Error())
 		}
 		q.Set("error_description", e.Message)
-		q.Set("error_code", strconv.Itoa(e.HTTPStatus))
```

Comment:
@hf @J0 do yall recall the reason for the setting the status code in the `error_code` query param?

Reply:
discussed internally and this is a bug that should be fixed